syntax="proto3";

message FullName {
    string name = 1;
    string surname = 2;
    optional string patronymic = 3; 
}

message Student {
    FullName fullname = 1;
    repeated uint32 marks = 2;
    float average_score = 3;
}

message StudentsGroup {
    repeated Student student = 1;
}
